initfiles/sbin/add_conf_settings.sh.in

```diff
@@ -20,9 +20,9 @@
 HPCC_CONFIG=${HPCC_CONFIG:-${CONFIG_DIR}/${ENV_CONF_FILE}}
 SECTION=${1:-DEFAULT}
 
-USER_NAME=`cat ${HPCC_CONFIG} | sed -n "/\[${SECTION}\]/,/\[/p" | grep "^user *= *" | sed -e 's/^user *= *//'`
+USER_NAME=`sed -n "/\[${SECTION}\]/,/\[/p" ${HPCC_CONFIG} | grep "^user *= *" | sed -e 's/^user *= *//'`
 
-PREFIX_PATH=`cat ${HPCC_CONFIG} | sed -n "/\[${SECTION}\]/,/\[/p" | grep "^path *= *" | sed -e 's/^path *= *//'`
+PREFIX_PATH=`sed -n "/\[${SECTION}\]/,/\[/p" ${HPCC_CONFIG} | grep "^path *= *" | sed -e 's/^path *= *//'`
```

Comment:
yes we still need those for the permissions in the sudoers file to be setup correctly.